### ISSUE TITLE
Keys.swift is missing

### ISSUE THREAD

**Issue by hamzaozturk** [nazmiyavuz/ConnectIst-FB-Clone#1]

**Comment by nazmiyavuz** [comment]
Yes I am aware of that. These projects are my first projects that's why I have wanted to simulate real real conditions, I saved in the gitignore folder.
Due to this conditions I added screen shots.
